website/docs/api/spancategorizer.mdx

```diff
@@ -89,6 +89,17 @@ architectures and their arguments and hyperparameters.
 | `negative_weight` <Tag variant="new">3.5.1</Tag>    | Multiplier for the loss terms. It can be used to downweight the negative samples if there are too many. It is only used when `add_negative_label` is `True`. Defaults to `1.0`. ~~float~~                                                                                                               |
 | `allow_overlap` <Tag variant="new">3.5.1</Tag>      | If `True`, the data is assumed to contain overlapping spans. It is only available when `max_positive` is exactly 1. Defaults to `True`. ~~bool~~                                                                                                                                                        |
 
+> ⚠️ Caution: if you set a non-default value for `spans_key`, you'll have to
```

Comment:
I think we have an infobox/warning `div`/accordion that we can use to emphasize this point (instead of just using an emoji).

Reply:
Done in https://github.com/explosion/spaCy/pull/13093/commits/cf5a99be41bbbcf34363b663c9a9be13ce90ecc1 and https://github.com/explosion/spaCy/pull/13093/commits/63cdc50680ba78ade746f84bd53049ce9ec6671e.